microcosm_flask/tests/conventions/test_crud.py

```diff
@@ -159,7 +159,7 @@ def test_search_with_context(self):
         })
 
     def test_reuse_search_self_link(self):
-        uri = "/api/address?list_param=a,b,c&enum_param=A".format(PERSON_ID_1)
```

Comment:
Unrelated changes, but those `format` were unnecessary, and mypy has gotten stricter and was erroring on them.

Reply:
I think we actually also have a better method of search here for query params (as an arg to the `get()` call below).

Reply:
We do. You caught me being lazy.